Error resolving plugin 'de.undercouch.download'

As mentioned on [Twitter](https://twitter.com/Coding4GaiAma/status/1228050223798374400)
I'm getting the following error
```
FAILURE: Build failed with an exception.

* Where:
Build file '/Users/Can/.pub-cache/hosted/pub.dartlang.org/git_bindings-0.0.11/android/build.gradle' line: 13

* What went wrong:
Error resolving plugin [id: 'de.undercouch.download', version: '4.0.0']
> Plugin request for plugin already on the classpath must not include a version

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 1s
Gradle task assembleDevDebug failed with exit code 1
Exited (sigterm)
```
just having the plugin in my `pubspec.yaml`, no imports nothing else and it'll crash.

<details>
<summary>Output of <code>flutter doctor -v</code></summary>
<pre><code>
[✓] Flutter (Channel stable, v1.12.13+hotfix.8, on Mac OS X 10.14.6 18G1012, locale de-PE)
    • Flutter version 1.12.13+hotfix.8 at /Users/Can/Applications/flutter
    • Framework revision 0b8abb4724 (4 weeks ago), 2020-02-11 11:44:36 -0800
    • Engine revision e1e6ced81d
    • Dart version 2.7.0

[✓] Android toolchain - develop for Android devices (Android SDK version 29.0.2)
    • Android SDK at /Users/Can/Library/Android/sdk
    • Android NDK location not configured (optional; useful for native profiling support)
    • Platform android-29, build-tools 29.0.2
    • ANDROID_HOME = /Users/Can/Library/Android/sdk
    • Java binary at: /Applications/Android Studio.app/Contents/jre/jdk/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 1.8.0_202-release-1483-b49-5587405)
    • All Android licenses accepted.

[✓] Xcode - develop for iOS and macOS (Xcode 11.3.1)
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Xcode 11.3.1, Build version 11C504
    • CocoaPods version 1.8.4

[✓] Android Studio (version 3.5)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin version 43.0.1
    • Dart plugin version 191.8593
    • Java version OpenJDK Runtime Environment (build 1.8.0_202-release-1483-b49-5587405)

[✓] VS Code (version 1.42.1)
    • VS Code at /Applications/Visual Studio Code.app/Contents
    • Flutter extension version 3.8.1

[✓] Connected device (2 available)
    • SM N950F            • 988ad045373353304f30 • android-arm64 • Android 9 (API 28)
    • AOSP on IA Emulator • emulator-5554        • android-x86   • Android 9 (API 28) (emulator)

• No issues found!
</code></pre>
</details>

What else could I provide?

**Update:** That happens on Android Emulator (Pixel 3a API 28 Android 9 Pie x86) and on my physical SM N950F (Note 8 running Android 9).
iOS simulator (don't have a physical) builds fine, `GitRepo.init(path)` fails with
```
[VERBOSE-2:ui_dart_state.cc(157)] Unhandled Exception: MissingPluginException(No implementation found for method gitInit on channel gitjournal.io/git)
#0      MethodChannel.invokeMethod (package:flutter/src/services/platform_channel.dart:319:7)
<asynchronous suspension>
#1      invokePlatformMethod (package:git_bindings/git_bindings.dart:13:20)
#2      GitRepo.init (package:git_bindings/git_bindings.dart:40:13)
#3      MainState.didChangeDependencies.<anonymous closure> (package:feder/main.dart:45:17)
<asynchronous suspension>
#4      _rootRunUnary (dart:async/zone.dart:1134:38)
#5      _CustomZone.runUnary (dart:async/zone.dart:1031:19)
#6      _FutureListener.handleValue (dart:async/future_impl.dart:139:18)
#7      Future._propagateToListeners.handleValueCallback (dart:async/future_impl.dart:680:45)
#8      Future._propagateToListeners (dart:async/future_impl.dart:709:32)
#9      Future._completeWithValue (dart:async/future_impl.dart:524:5)
#10     _AsyncAwaitCompleter.complete (dart:async-patch/as<…>
```

**update** just updated `minSdkVersion` from `19` to `21` as GitJournal uses, without success tho

--- a/android/build.gradle
+++ b/android/build.gradle
@@ -10,7 +10,7 @@ buildscript {
 }
 
 plugins {
-    id "de.undercouch.download" version "4.0.4"
+    id "de.undercouch.download"
 }
 
 group 'io.gitjournal.git_bindings'

